src/detail-page/BackButton.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import styled from 'styled-components';
+import Typography from '@material-ui/core/Typography';
+import ArrowBackIosIcon from '@material-ui/icons/ArrowBackIos';
+import { Button } from '@material-ui/core';
+import { useHistory } from 'react-router-dom';
+
+const BackButtonStyle = styled(Button)`
+  margin: 1em;  
```

Comment:
I think it would look nice, if the button was aligned to the left with tabs container, so maybe this margin could be on the top and bottom only?

Reply:
I think it's done